config/install/migrate_plus.migration.example_migration_node.yml

```diff
@@ -166,3 +166,8 @@ process:
 destination:
   plugin: 'entity:node'
   default_bundle: islandora_object
+
+migration_dependencies:
+  required:
+    - migrate_plus.migration.example_migration_file
```

Comment:
We don't need to make the node migration dependent on the file migration. It doesn't matter which of the two run first, just so long as they both run before the media one.

Reply:
Good point.

Reply:
@seth-shaw-unlv done. When debugging I controlling the firing make troubleshooting "seem" easier for me. lol. Anyways, it's ready for testing.